app/services/data/insert-deduction.js

```diff
@@ -0,0 +1,24 @@
+const config = require('../../../knexfile').intweb
+const knex = require('knex')(config)
+
+module.exports = function (claimId, deductionType, amount) {
```

Comment:
take claimDeduction domain object for param, `module.exports = function (claimId, claimDeduction) {`